packages/ui-library/src/components/notifications/index.js

```diff
@@ -151,6 +151,7 @@ const notificationsClassNameMap = {
 		"bottom-center": "yst-notifications--bottom-center",
 		"bottom-left": "yst-notifications--bottom-left",
 		"top-center": "yst-notifications--top-center",
+		inherit: "yst-notifications--inherit",
```

Comment:
Hmm, interesting. Does this have a use-case outside of the stories I wonder.
Maybe move the `yst-fixed` to the other classes and remove the `inherit` enforcing?
That way you can do away with the wrapper in the Template too

Something like this:
```css
       .yst-notifications {
            @apply yst-w-full yst-z-20 yst-pointer-events-none yst-flex yst-flex-col yst-space-y-4;
        }

        .yst-notifications--bottom-center {
            @apply yst-fixed yst-bottom-8 yst-items-center;
        }

        .yst-notifications--bottom-left {
            @apply yst-fixed yst-bottom-8 yst-left-8;
        }

        .yst-notifications--top-center {
            @apply yst-fixed yst-top-8 yst-items-center;
        }
```